### DISCUSSION TITLE
Github sponsors: Stripe account countries

### DISCUSSION THREAD

**Discussion by bennymeg** [community/community#28397] [Other Feature Feedback, Questions, & Ideas] [answered]
I have just received an invitation to Github sponsors.
I see Github sponsors is [only] using Stripe as a fiscal host.
If Stripe is not available in my country yet, would I be still able to open a dedicated Stripe account through Github?

**Comment by bennymeg** [comment]
Yes,
Github is using `Stripe Connect`, rather then regular Stripe account, which have a different, larger supported countries list.
If you received an invitation to Github sponsors, it probably means your country is supported.
You can verify your country is supported [here](https://stripe.com/connect/pricing#).

**Comment by jfullstackdev** [comment]
yes, that's all handled by GitHub, even the Philippines is not fully supported by Stripe but GitHub Sponsors already handled it

**Comment by jlord** [comment, marked as answer]
Hiya! Sponsors is using Stripe Connect with Express accounts which supports payouts to more regions than if one was signing up to Stripe to receive payments. See this [list of cross-border payouts](https://stripe.com/docs/connect/cross-border-payouts) for a more accurate list.
